sql/core/src/test/scala/org/apache/spark/sql/BenchmarkQueryTest.scala

```diff
@@ -63,11 +63,17 @@ abstract class BenchmarkQueryTest extends QueryTest with SharedSparkSession {
 
   protected def checkGeneratedCode(plan: SparkPlan, checkMethodCodeSize: Boolean = true): Unit = {
     val codegenSubtrees = new collection.mutable.HashSet[WholeStageCodegenExec]()
-    plan foreach {
-      case s: WholeStageCodegenExec =>
-        codegenSubtrees += s
-      case _ =>
+
+    def findSubtrees(paln: SparkPlan): Unit = {
```

Comment:
Since this fix is not related to `EXPLAIN`, could you update the title and the PR description?